README.md

```diff
@@ -242,12 +242,22 @@ achieve this meaningful diff the tool must do perform two operations:
 
 ### Workflow Description
 
-The 2 most common cases
+To Compare a known valid reference configuration with a live cluster:
+
+`kubectl cluster-compare -r <referenceConfigurationDirecotry>`
 
 To Compare a known valid reference configuration with a local set of CRs:
 
 `kubectl cluster-compare -r <referenceConfigurationDirecotry> -f <inputConfiguration>`
 
+To Compare a known valid reference configuration with a live cluster and with a user config:
```

Comment:
Because the tool is comparing a reference with the configuration on a cluster, I think we need to clarify the term "user config" here to make it obvious that this is a file which is tuning the behavior of the tool and not part of what is being compared.